conn.go

```diff
@@ -377,13 +378,16 @@ func (c *Conn) handleIncomingPacket(buf []byte) error {
 
 	switch content := r.content.(type) {
 	case *alert:
+		c.log.Trace("<- Alert")
 		if content.alertDescription == alertCloseNotify {
 			return c.Close()
 		}
 		return fmt.Errorf("alert: %v", content)
 	case *changeCipherSpec:
+		c.log.Trace("<- ChangeCipherSpec")
 		c.setRemoteEpoch(c.getRemoteEpoch() + 1)
 	case *applicationData:
+		c.log.Trace("<- ApplicationData")
```

Comment:
I would avoid this trace since this will be an extremely hot code path in more demanding applications.

Reply:
I did not know when will go to this path

Reply:
If I do not enable the trace. Are There will be  any performance issue？